### ISSUE TITLE
Implement builtin `Keyword<P>`

### ISSUE THREAD

**Issue by phorward** [tokay-lang/tokay#121]
This origins from #105 to replace *tokay.tok*s `'kw' _standalone_` by `Keyword<'kw'>`, which is also useful for other use-cases and languages.

**Comment by phorward** [comment]
Implementation draft!

```tokay
# Parse keywords, which may not be the prefix of another identifier.
Keyword : @<P> {
    P Not<Alphabetic>
}

Keyword<'if'> print("Real If!")
print("Skipping" Char)
```

**Comment by phorward** [comment]
fixed by #125
